bootstrap.sh

```diff
@@ -392,7 +260,7 @@ install_sentry_cli() {
     # This ensures that sentry-cli has a directory to install under
     [ ! -d /usr/local/bin ] && (
       sudo_askpass mkdir /usr/local/bin
-      sudo_askpass chown ${USER}:admin /usr/local/bin
```

Comment:
Is there a reason we shouldn't set the owner to the admin group? Or is that the default behavior when undefined?